docs/ja/user-guide/configuration/index.md

```diff
@@ -99,4 +117,5 @@ toc:
             <p>コマンドラインで入力するように、ビルドで実行されるコマンドのリストを定義します。同じジョブ内では、環境変数はステップ間で受け渡されます。ステップの定義は全てのジョブに必須です。Screwdriverのステップとして予約語になっているため、ステップ名を'sd-'で始めることはできません。Screwdriverは'/bin/sh'をステップ実行のために使用するため、異なるターミナルやシェルを使用すると予期せぬ振る舞いをするかもしれません。</p>
         </div>
     </div>
-</div>
+</code></pre>
+<div data-md-type="block_html"></div>
```

Comment:
Probably, this change isn't unnecessary.

Reply:
removed.